core/pta/stats.c

```diff
@@ -152,6 +184,13 @@ static TEE_Result invoke_command(void *psess __unused,
 		return get_alloc_stats(ptypes, params);
 	case STATS_CMD_MEMLEAK_STATS:
 		return get_memleak_stats(ptypes, params);
+	case STATS_CMD_TA_STATS:
+#if defined(CFG_TA_STATS)
+		return get_user_ta_stats(ptypes, params);
+#else
+		EMSG("STATS_CMD_TA_STATS not supported\n");
+		return TEE_ERROR_NOT_SUPPORTED;
```

Comment:
move this into `get_user_ta_stats()`? E.g. at function entry:
```c
	if (!IS_ENABLED(CFG_TA_STATS)) {
		EMSG("STATS_CMD_TA_STATS not supported\n");
		return TEE_ERROR_NOT_SUPPORTED;
	}

Reply:
yeah... code will be more neat in this way...will do.